.github/workflows/InferenceSystem.yaml

```diff
@@ -223,6 +223,17 @@ jobs:
       run: |
         docker build . -t live-inference-system -f ./Dockerfile
 
+    - name: Test docker container
+      working-directory: InferenceSystem
+      run: |
+        docker run --rm \
+          --env-file .env \
+          -v $PWD/config:/config \
+          live-inference-system \
+          python3 -u ./src/LiveInferenceOrchestrator.py \
+          --config /config/Test/FastAI_LiveHLS_OrcasoundLab.yml \
+          --max_iterations 2
```

Comment:
Does this do anything useful? Github Actions won't be able to tell if the container crashes immediately.